src/js/tabs/setup.js

```diff
@@ -369,29 +370,39 @@ setup.initialize = function (callback) {
 
             if (supported && ispConnected()) {
                 const buildRoot = `https://build.betaflight.com/api/builds/${FC.CONFIG.buildKey}`;
+
```

Comment:
Added a few blank lines for separation (consistent with other parts of the file).